packages/neuron-wallet/src/services/settings.ts

```diff
@@ -84,16 +80,34 @@ export default class SettingsService extends Store {
         ckbDataPath: path.resolve(app.getPath('userData'), 'chains/mainnet'),
       })
     )
-    if (!this.ckbDataPath) {
-      this.ckbDataPath = path.resolve(app.getPath('userData'), 'chains/mainnet')
-    }
-    if (!this.testnetLightDataPath) {
-      this.testnetLightDataPath = path.resolve(app.getPath('userData'), 'chains/light/testnet')
+    if (!this.getNodeDataPath(LIGHT_CLIENT_MAINNET)) {
+      this.migrateDataPath()
     }
   }
 
   private onLocaleChanged = (lng: Locale) => {
     BrowserWindow.getAllWindows().forEach(bw => bw.webContents.send('set-locale', lng))
     updateApplicationMenu(null)
   }
+
+  migrateDataPath() {
+    const networkChain = NetworksService.getInstance().getCurrent()?.chain
+    const currentCkbDataPath = this.readSync(settingKeys.ckbDataPath)
+    this.writeSync(`${settingKeys.nodeDataPath}_${networkChain}`, currentCkbDataPath)
+    const defaultMainNetworkDir = path.resolve(app.getPath('userData'), 'chains/mainnet')
+    if (networkChain === 'ckb') {
+      this.writeSync(`${settingKeys.nodeDataPath}_ckb_testnet`, path.resolve(app.getPath('userData'), 'chains/testnet'))
```

Comment:
Is this still necessary when full node testnet is not included in this app

Reply:
Only the `else` branch is necessary, the `ckb_testnet` data path can be removed. I will remove it.